src/firmware_main_loop.c

```diff
@@ -47,7 +73,19 @@ void firmware_main_loop(void)
 #endif
 
     while (1) {
-        // Do USB I/O
+        // Do UART I/O
+        if (!_usb_hww_request_seen) {
```

Comment:
also check that this is the bb02+ platform - currently this PR does not work on a regular bb02 because of it.

Reply:
done